src/services/userService.js

```diff
@@ -0,0 +1,44 @@
+const { Op } = require('sequelize');
+import User from '../models/user';
+import { v4 as uuidv4 } from 'uuid';
+
+class UserService {
+    getAllUsers() {
+        return User.findAll();
+    }
+
+    getUserById(id) {
+        return User.findByPk(id);
+    }
+
+    createUser(userDTO) {
+        return User.create({ id: uuidv4(), ...userDTO, isDeleted: false });
```

Comment:
You can assign uuid directly in model, then you won't need to remember to do it all the time in the code

Reply:
Assigned default id value in model, will remove id assignment in Service.